src/screens/EtherspotDeploymentInterjection/EtherspotDeploymentInterjection.js

```diff
@@ -127,6 +179,22 @@ function EtherspotDeploymentInterjection() {
               btnTextStyle={styles.btnTextStyle}
             />
           </ButtonContainer>
+          {!gasInfo.isFetched && <SpinnerWrapper><Spinner size={20} trackWidth={2} /></SpinnerWrapper>}
+          {!!deploymentFee && (
+            <Tooltip
```

Comment:
Visually tooltip looks like attached more to button then fee text. 
![image](https://user-images.githubusercontent.com/6368606/129890613-24c0a458-4ff1-4c19-bd5e-8284acb0b362.png)

Reply:
Will fix.